### PR TITLE
Feature/7119 create datamodels repository

### PR DESCRIPTION
**Pull request by RonnyB71**
Continuing to play around with CQRS pattern in this tool.

### CODE REVIEW

File: RepoCleanup/Functions/SharedFunctionSnippets.cs [hunk 1]
```diff
@@ -71,6 +71,80 @@ public static bool ShouldThisApplyToAllOrgs()
...
+                Console.Write("\r\nSet website for org: ");
+                website = Console.ReadLine();
+
+                if (string.IsNullOrEmpty(website))
+                {
+                    isValid = true;
+                }
+                else
+                {
+                    isValid = Regex.IsMatch(website, "^[a-zA-Z0-9\\-._/:]*$");
```
Comment (SandGrainOne) [inline RepoCleanup/Functions/SharedFunctionSnippets.cs:108]: Should we allow Norwegian letters?

**Review by SandGrainOne — APPROVED** [review]
Approved with comments

File: RepoCleanup/RepoCleanup.csproj [hunk 2]
```diff
@@ -23,4 +23,8 @@
     </None>
   </ItemGroup>
 
+  <ItemGroup>
```
[the remark this answers is not in the document]
Comment (RonnyB71) [reply RepoCleanup/RepoCleanup.csproj:26]: Thx, removed.

File: RepoCleanup/Functions/SharedFunctionSnippets.cs (hunk 1 above)
Comment (RonnyB71) [reply RepoCleanup/Functions/SharedFunctionSnippets.cs:108]: I know it's allowed on host names, but is it recommended? I haven't changed this part, just moved it, so it's always been like this. Perhaps consider it if it becomes a issue?

**Pull request merged**